Martinh/demo sdk version (#546)
wormhole-foundation/wormhole-docs: products/token-bridge/get-started.md

    npm init -y
    ```

> 2. Install the required dependencies. In this guide, we will use the TypeScript SDK version `2.4.0`:

Reviewer: ```suggestion
2. Install the required dependencies. In this guide, you’ll use the TypeScript SDK version `2.4.0`:
```
